app/assets/javascripts/admin/controllers/admin-web-hooks-show.js.es6

```diff
@@ -0,0 +1,100 @@
+import { popupAjaxError } from 'discourse/lib/ajax-error';
+import computed from 'ember-addons/ember-computed-decorators';
+
+export default Ember.Controller.extend({
+  needs: ['adminWebHooks'],
+  eventTypes: Em.computed.alias('controllers.adminWebHooks.eventTypes'),
+  defaultEventTypes: Em.computed.alias('controllers.adminWebHooks.defaultEventTypes'),
+  contentTypes: Em.computed.alias('controllers.adminWebHooks.contentTypes'),
+
+  @computed('model.isSaving', 'saved', 'saveButtonDisabled')
+  savingStatus(isSaving, saved, saveButtonDisabled) {
+    if (isSaving) {
+      return I18n.t('saving');
+    } else if (!saveButtonDisabled && saved) {
+      return I18n.t('saved');
+    }
+    this.set('saved', false);
```

Comment:
I am using the side effect of validation process to reset promoting text. If the model is saved once, the `Saved!` can be clear after some change is made in the form.